modules/tinymce/CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - The `caption`, `address` and `dt` elements were allowed to have non-inline children elements when the editor schema was set to `html4`. #TINY-9768
+- svg icons for back and foreground colors now uses class instead of id to identify svg elements that should change color #TINY-9844
```

Comment:
Not sure if we should use the upper case abbreviation here and added a dot after the sentence. 

```suggestion
- SVG icons for back and foreground colors now uses class instead of id to identify SVG elements that should change color. #TINY-9844
```